docs/operators.md

```diff
@@ -14,9 +14,9 @@ An operator consists, broadly, of collections of runnable controllers, one type
 but a user can easily extend this by having a new controller which implements the `operator.Controller` interface.
 
 The controller offered by the SDK is the `operator.InformerController`, which is a controller that is composed of two sets of objects:
```

Comment:
This says "two sets" but we're enumerating three different ones now.

Reply:
ugh missed that, fixed!